dev-util/claude-code/claude-code-1.0.6.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Claude Code - an agentic coding tool by Anthropic"
+HOMEPAGE="https://www.anthropic.com/claude-code"
+SRC_URI="https://registry.npmjs.org/@anthropic-ai/claude-code/-/claude-code-${PV}.tgz"
+S="${WORKDIR}/package"
+
+# NOTE(JayF): claude-code is only usable via paid subscription and has a
+#             clickthrough EULA-type license. Please see $HOMEPAGE for
+#             full details.
+LICENSE="all-rights-reserved"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+QA_PREBUILT="usr/lib64/node_modules/@anthropic-ai/claude-code/vendor/*"
+RESTRICT="strip"
+
+RDEPEND="
+	>=net-libs/nodejs-18
+	sys-apps/ripgrep
+"
+BDEPEND=">=net-libs/nodejs-18[npm]"
+
+DOCS=( README.md )
+
+src_compile() {
+	# Skip, nothing to compile here.
+	:
+}
+
+src_install() {
+	local -a my_npm_opts=(
+		--audit false
+		--color false
+		--foreground-scripts
+		--global
+		--offline
+		--omit dev
+		--prefix "${ED}/usr"
+		--progress false
+		--verbose
+	)
+	npm "${my_npm_opts[@]}" install "${DISTDIR}/${P}.tgz" || die "npm install failed"
```

Comment:
If it reads that tarball directly, you may want to define a stub `src_unpack` to not waste time unpacking it unnecessarily.

Reply:
You may also want to use `edo` here instead which would let you drop the `|| die` and get a more verbose log.